geemap/toolbar.py

```diff
@@ -27,211 +31,137 @@
 
 
 @map_widgets.Theme.apply
-class Toolbar(widgets.VBox):
-    """A toolbar that can be added to the map."""
-
-    @dataclass
-    class Item:
-        """A representation of an item in the toolbar.
+class ToolbarItem(anywidget.AnyWidget):
+    """A toolbar item widget for geemap."""
+
+    _esm = pathlib.Path(__file__).parent / "static" / "toolbar_item.js"
+    active = traitlets.Bool(False).tag(sync=True)
+    icon = traitlets.Unicode("").tag(sync=True)
+    # Unfortunately, "tooltip" is a property already defined on ipywidgets.
```

Comment:
Maybe, ""tooltip" is already defined on ipywidgets.Widget".

Reply:
Done.